aeacus-src/release_windows.go

```diff
@@ -2,63 +2,66 @@ package main
 
 func writeDesktopFiles(mc *metaConfig) {
 	firefoxBinary := `C:\Program Files (x86)\Mozilla Firefox\firefox.exe`
-	if mc.Cli.Bool("v") {
+	if verboseEnabled {
 		infoPrint("Writing ScoringReport.html shortcut to Desktop...")
 	}
 	cmdString := `$WshShell = New-Object -comObject WScript.Shell; $Shortcut = $WshShell.CreateShortcut("C:\Users\` + mc.Config.User + `\Desktop\ScoringReport.lnk"); $Shortcut.TargetPath = "` + firefoxBinary + `"; $Shortcut.Arguments = "C:\aeacus\web\ScoringReport.html"; $Shortcut.Save()`
 	shellCommand(cmdString)
-	if mc.Cli.Bool("v") {
+	if verboseEnabled {
 		infoPrint("Writing ReadMe.html shortcut to Desktop...")
 	}
 	cmdString = `$WshShell = New-Object -comObject WScript.Shell; $Shortcut = $WshShell.CreateShortcut("C:\Users\` + mc.Config.User + `\Desktop\ReadMe.lnk"); $Shortcut.TargetPath = "` + firefoxBinary + `"; $Shortcut.Arguments = "C:\aeacus\web\ReadMe.html"; $Shortcut.Save()`
 	shellCommand(cmdString)
-	if mc.Cli.Bool("v") {
+	if verboseEnabled {
 		infoPrint("Creating or emptying TeamID.txt file...")
 	}
 	cmdString = "echo 'YOUR-TEAMID-HERE' > C:\\aeacus\\TeamID.txt"
 	shellCommand(cmdString)
-	if mc.Cli.Bool("v") {
+	if verboseEnabled {
 		infoPrint("Writing TeamID shortcut to Desktop...")
 	}
 	cmdString = `$WshShell = New-Object -comObject WScript.Shell; $Shortcut = $WshShell.CreateShortcut("C:\Users\` + mc.Config.User + `\Desktop\TeamID.lnk"); $Shortcut.TargetPath = "C:\aeacus\phocus.exe"; $Shortcut.Arguments = "-i yes"; $Shortcut.Save()`
 	shellCommand(cmdString)
 
-	// todo configure autologin user (netplwiz?)
 	// domain compatibility? doubt
 }
 
-func installService(mc *metaConfig) {
-	if mc.Cli.Bool("v") {
+func configureAutologin(mc *metaConfig) {
```

Comment:
Can we keep unfinished functions on TODO instead of just empty?

Reply:
I could go either way, I kind of like having the framework/skeleton there since it makes it easier to start working on them later. But yeah if you think it's cluttering it up we can remove it.